### ISSUE TITLE
UnusedParameter false-positive with string interpolation

### ISSUE THREAD

**Issue by EvgenyOrekhov** [slevomat/coding-standard#421]
```php
class C
{
    public function f($p)
    {
        return "${p}";
    }
}
```

Expected result: no warnings

Actual result:

> Unused parameter $p.

Slevomat Coding Standard version: 4.7.0

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/68fe9facd226a9d576ced4a1510e7109d943ade6
